### ISSUE TITLE
Superimposing two histograms from different runs?

### ISSUE THREAD

**Issue by errhernandez** [tensorflow/tensorboard#6206]
Please describe the problem that you’re facing and the enhancements that
you’d like to see. Feel free to include screenshots or code samples.
Hello. It would be nice to be able to superimpose on the same graph histograms of predictions on a test set made with different hyperparameters; I can see that these can be plotted side by side, but for a more detailed comparison it would be desirable to have them on the same plot, just like it happens automatically for loss/accuracy for the different runs. It might be that this can be done already, but if so, I haven't been able to see how...
Thanks!

**Comment by rileyajones** [comment]
Are you using the Time Series dashboard or the Histograms dashboard?

Can you give a deeper explanation of how you would expect this to work?
* How would you imagine selecting the runs that should be plotted?
* Would you imagine adding a run to an existing card? Or would expect a new card to appear?
* Histograms cards often have many histograms, how would you differentiate the runs? 
![image](https://user-images.githubusercontent.com/78179109/220416914-e8bcac1f-3b23-43eb-aab3-cb3ad8f4931d.png)

**Comment by errhernandez** [comment]
Hello; thanks for your reply. See below for my answers to your questions:

> Are you using the Time Series dashboard or the Histograms dashboard?

I am using the Histograms dashboard. 

> Can you give a deeper explanation of how you would expect this to work?

At the end of every run with same model but different hyper-parameters, say, I save to the run file a histogram of errors, prediction - target. It would be nice to be able to plot on the same graph a selection of these histograms for a more direct comparison than is possible by simply plotting side by side in the Histograms dashboard. Ideally, one should be able to click which particular run results to plot, just like one does in the scalars dashboard. 

> * How would you imagine selecting the runs that should be plotted?

See above: by clicking on the appropriate run labels like one does in the scalars dashboard, or by pinning the desired graphs directly in the histograms dashboard. 

> * Would you imagine adding a run to an existing card? Or would expect a new card to appear?

By "card" you mean a particular histogram? That would also be a possible way, yes.

> * Histograms cards often have many histograms, how would you differentiate the runs?
>   ![image](https://user-images.githubusercontent.com/78179109/220416914-e8bcac1f-3b23-43eb-aab3-cb3ad8f4931d.png)

If one can pick and choose which runs to plot, this would not be an issue, right? The user could focus on the desired ones. To be clear: I don't mean plotting histograms as in the time series that you show in your figure, but simply plotting two (or more) histograms from different runs on the same graph. 

I hope this answers your questions and thanks for your response.
